dockerfiles/init/modules/openshift/files/scripts/che-spi-openshift.yml

```diff
@@ -87,6 +87,7 @@ items:
     CHE_INFRA_OPENSHIFT_OAUTH__TOKEN: ""
     CHE_INFRA_OPENSHIFT_TRUST__CERTS: "false"
     CHE_INFRA_OPENSHIFT_TLS__ENABLED: "true"
+    CHE_INFRA_OPENSHIFT_PVC_STRATEGY: "common"
```

Comment:
Please, add an ability to set another strategy in deployment script

Reply:
ok